src/algorithm/index.ts

```diff
@@ -49,17 +49,22 @@ const algo2c4 = createPredict(
   })
 );
 
+// Set of endpoints returned when calling request for company 3 or company 4 algorithms.
```

Comment:
Extracting these objects allows the unit tests to spy on them and validate the input we send to the endpoint.